src/commands/extension/watch.ts

```diff
@@ -24,6 +24,7 @@ export default class Create extends BaseCommand {
   performingInstall = false;
 
   async run() {
+    await this.performInstall();
```

Comment:
Rather than this, it would be possible to remove the `ignoreInitial: true` in the `chokidar.watch`. However, that creates an additional message that's not that appealing:

```
Watching for changes in the current directory ...
... detected 7100 files changed
Installing extension '@aha-develop/trello-import' to 'http://reallybigaha.lvh.me:3000'
   contributes importers: 'cards'
Compiling... done
Uploading... done
```

Any way I could think of to avoid that would turn out messier than just calling `performInstall` manually first.